### PR TITLE
Update master references to main

### PR DESCRIPTION
**Pull request by CaseyHillers**

### CODE REVIEW

File: app_dart/lib/src/service/buildbucket.dart [hunk 1]
```diff
@@ -70,6 +70,7 @@ class BuildBucketClient {
     }
 
     httpRequest.write(json.encode(request.toJson()));
+    print(json.encode(request.toJson());
```
Comment (yusufm) [inline app_dart/lib/src/service/buildbucket.dart:73]: Print a description of whats being output here?

**Review by keyonghan — APPROVED** [review]
LGTM

File: app_dart/lib/src/service/buildbucket.dart (hunk 1 above)
Comment (godofredoc) [reply app_dart/lib/src/service/buildbucket.dart:73]: Would print show in the remote logs?

Comment (CaseyHillers) [reply app_dart/lib/src/service/buildbucket.dart]: Removed, this was from a mock checkout

Prints do show in stackdriver, but they're not properly correlated (it's very useful if you don't want to plumb logging instances to somewhere like here)

**Review by godofredoc — APPROVED** [review]

**Comment by CaseyHillers** [comment]
Submitting as the checks are expected to fail, and this is blocking the null safety migration

File: app_dart/lib/src/service/buildbucket.dart (hunk 1 above)
Comment (godofredoc) [reply app_dart/lib/src/service/buildbucket.dart]: Good to know, thanks!

**Pull request merged**